test/fix-me.js

```diff
@@ -1,5 +1,14 @@
-var expect = require('chai').expect
-var fixMe = require('../lib/fix-me.js')
+var expect = require('chai').expect,
+    fs = require('fs'),
+    intercept = require("intercept-stdout"),
+    captured_text = "",
+    unhook_intercept = intercept(function(txt) {
```

Comment:
Declaring this here means that it will capture anything between now and when you run `unhook_intercept()`. Can you move the declaration of `captured_text` and `unhook_intercept` into the test itself?

Also, can you camelcase the variable names?